### ISSUE TITLE
Helm chart deployment broken since version 2.5.1

### ISSUE THREAD

**Issue by muellermatthias** [pulumi/pulumi-kubernetes#1321]
### Problem description

We are using the Kubernetes provider to deploy the nginx-ingress helm chart. Since the version 2.5.1 the deployment always fails with the error below. We only updated the kubernetes provider version so we are sure it's caused by the update.

Apart from that we are using AKS 1.16.13. To be sure I also deployed the helm chart manually via helm, which works.

### Errors & Logs
```
kubernetes:rbac.authorization.k8s.io:ClusterRoleBinding (test-aksref-ingress-controller-nginx-ingress):
    error: resource test-aksref-ingress-controller-nginx-ingress was not successfully created by the Kubernetes API server : ClusterRoleBinding.rbac.authorization.k8s.io "test-aksref-ingress-controller-nginx-ingress" is invalid: subjects[0].namespace: Required value
```
### Affected product version(s)

@Pulumi/Kubernetes: >=2.5.1

### Reproducing the issue

Here is our code for the helm deployment:

```
        new k8s.helm.v3.Chart(`${name}-ingress-controller`, {
            chart: "nginx-ingress",
            version: "1.41.2",
            fetchOpts: {
                repo: "https://kubernetes-charts.storage.googleapis.com",
            },
            values: {
                controller: {
                    service: {
                        loadBalancerIP: "10.10.10.10,
                        annotations: {
                            "service.beta.kubernetes.io/azure-load-balancer-internal": "true"
                        }
                    },
                }
            },
        }, {
            provider: cluster, // we're using the admin credentials here
        })
```

**Comment by lblackstone** [comment]
Thanks for the detailed error report! I'm looking into this now.

Not sure if this is just a copy-paste error in the issue, but `loadBalancerIP: "10.10.10.10,` is missing a closing `"` character.

**Comment by lblackstone** [comment]
Ok, was able to repro the issue locally, and am getting the following diff (which explains the error you're seeing):
```diff
          ~ subjects: [
              ~ [0]: {
                      - namespace: "default"
                    }
            ]
```

I'll get this fixed ASAP

**Comment by muellermatthias** [comment]
@lblackstone The missing `"` was my fault. It's correct in my code.

Thanks for looking into this.

**Comment by lblackstone** [comment]
This should be fixed in the v2.6.1 release, which will be published shortly.
